autorag/nodes/passagereranker/colbert.py

```diff
@@ -26,11 +26,14 @@ def colbert_reranker(queries: List[str], contents_list: List[List[str]],
     :param top_k: The number of passages to be retrieved
     :param batch: The number of queries to be processed in a batch
         Default is 64.
+        The batch size must always be greater than 1.
     :param model_name: The model name for Colbert rerank.
         You can choose colbert model for reranking.
         Default is "colbert-ir/colbertv2.0".
     :return: Tuple of lists containing the reranked contents, ids, and scores
     """
+    assert batch > 1, "Batch size must be greater than 1"
+
```

Comment:
Why the batch size must be greater than 1?

Reply:
I didn't fully realize torch, so I don't know why the score is different only when the batch is 1, but it's a hotfix, so I temporarily handled it with assert like this.

Still, I was able to handle issue #301 with the flatten method above.

As I parallelize the UPR (PR #334) and learn more about torch operations, I'll fix issues that don't work in batch 1 only!

Once this PR is merged, I'll create another issue!